tests/api/handlers/test_admin.py

```diff
@@ -11,3 +16,104 @@ def test_admin_rules_forbidden(self, client, api_identity):
         api_identity.admin = False
         response = client.get(f"{self.path}/rules")
         assert response.status_code == status.HTTP_403_FORBIDDEN
+
+    def test_list_disabled_rules(self, client, api_identity, db_rule, db_rule_disabled):
+        api_identity.admin = True
+
+        response = client.get(f"{self.path}/rules")
+        assert response.status_code == status.HTTP_200_OK
+        result = response.json()
+        assert len(result) == 1
+        assert result[0] == {
+            "id": 2,
+            "name": "disabledrule",
+            "disabled": True,
+            "user": {"id": 1, "name": "testuser", "is_admin": False},
+            "tracking_rule": {"name": "users-followed", "params": ["user1", "user2"]},
+            "generation_rules": [
+                {
+                    "destinations": [{"protocol": "email", "address": "dude@mcpants"}],
+                    "filters": {
+                        "applications": [],
+                        "severities": [],
+                        "topic": None,
+                        "my_actions": False,
+                    },
+                }
+            ],
+            "generated_last_week": 0,
+        }
+
+    @mock.patch("fmn.api.handlers.admin.publish")
```

Comment:
You can use `fedora_messaging.testing.mock_sends()` instead, see other tests for a usage example.